requirements.txt

```diff
@@ -3,3 +3,4 @@ xlwt
 xlrd
 xlsxwriter
 pyyaml
+future
```

Comment:
Hmm...  Yes, this should be added, but it points out a separate issue.  I thought `requirements.txt` was mostly for specifying specific versions.  Such as using `pip freeze > requirements.txt`.  Though I'll admit that in my other projects I only enter direct dependencies and their versions into `requirements.txt`.  :[  I should stop doing that.

Reply:
Agree, I do not think this should be solve in this PR :)